src/domains/wallet-common/03_storage.tf

```diff
@@ -70,18 +70,6 @@ module "wallet_storage" {
   tags = var.tags
 }
 
-resource "azurerm_storage_queue" "wallet_usage_update_queue" {
-  name                 = "${local.project}-usage-update-queue"
-  storage_account_name = module.wallet_storage.name
-}
-
-//storage queue for blue deployment
-resource "azurerm_storage_queue" "wallet_usage_update_queue_blue" {
-  count                = var.env_short == "u" ? 1 : 0
-  name                 = "${local.project}-usage-update-queue-b"
-  storage_account_name = module.wallet_storage.name
-}
-
```

Comment:
revert this change: this domain is no more maintained and mark for removal, no need to modify it

Reply:
done here b091def87b44a4f0aa85d6628a5f34c5408657c2